lib/CodeGen/PagerandoBinning.cpp

```diff
@@ -75,8 +75,10 @@ bool PagerandoBinning::runOnModule(Module &M) {
 
   for (auto &F : M) {
     const MachineFunction &MF = MMI.getMachineFunction(F);
-    unsigned Bin = F.isRandPage() ? AssignToBin(MF) : DefaultBin;
-    MMI.setBin(&F, Bin);
+    if (F.isRandPage()) {
+      unsigned Bin = AssignToBin(MF);
+      F.setSectionPrefix(".page" + utostr(Bin));
```

Comment:
The only uses I could find are `.hot`/`.cold` which could be used by the linker for optimization but not correctness.